x-pack/plugins/security_solution/public/overview/components/overview_risky_host_links/risky_hosts_panel_view.tsx

```diff
@@ -30,6 +30,7 @@ const columns: Array<EuiTableFieldDataColumnType<LinkPanelListItem>> = [
     align: 'right',
     field: 'count',
     name: 'Risk Score',
+    render: (riskScore) => (isNaN(riskScore) ? riskScore : Number.parseFloat(riskScore).toFixed(2)),
```

Comment:
I am not sure what value can be in `riskScore`, but the isNaN will return `false` if  `riskScore` will be a `null` or `''`.
And `Number.parseFloat(null)` will be NaN.

Here are more examples of this `isNaN` behaviour
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isNaN#examples

So maybe, we want to use  `Number.isNaN` instead